tools/unreal-vat-demo/Content/Python/build_vat_demo.py

```
@@ -0,0 +1,453 @@
+    mat  = unreal.load_asset(DEMO_DIR + "/M_OpenVAT")
+    if mesh is None or mat is None:
+        unreal.log_error("Missing SK_Rumba or M_OpenVAT — can't spawn.")
+        return None
+
+    # Delete any pre-existing OpenVATDancer so reruns don't pile up.
+    for a in actor_lib.get_all_level_actors():
+        try:
+            if a and a.get_actor_label() == "OpenVAT_Dancer":
+                actor_lib.destroy_actor(a)
+        except Exception:
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)